versioned_docs/version-24.10/api/rest-api-v1.md

```diff
@@ -12,7 +12,12 @@ the Centreon command line API.
 
 ## Permissions
 
-To perform API calls, you must be an administrator.
+Users can be authorized to perform API calls by configuring the corresponding fields located in the [**Centreon Authentication**](../monitoring/basic-objects/contacts-create.md#centreon-authentication) tab.
+- The [**Configuration API**](#configuration) can only be called by administrators. Note that this field only applies to the v2 API.
```

Comment:
for APIv2, only administrator can access. For v2, all user having access to configuration with this option enabled can access.